src/app/cart/cart.tpl.html

```diff
@@ -32,7 +32,7 @@ <h3 class="panel-name" translate>Your Gift Cart</h3>
                         <span class="giftsum-frequency giftsum-title">{{i.frequency}}</span>
                         <span class="giftsum-date giftsum-detail" ng-if="i.frequency !== 'Single'">
                           <span translate>Starts on:</span>
-                          {{i.giftStartDate | date:'mediumDate'}}
+                          {{i.giftStartDate.format('LL')}}
```

Comment:
I believe this should be 'll'.
From Angular Docs
> 'longDate': equivalent to 'MMMM d, y' for en_US locale (e.g. September 3, 2010)
> 'mediumDate': equivalent to 'MMM d, y' for en_US locale (e.g. Sep 3, 2010)

From Moment Docs
> 'LL' September 4 1986
> 'll' Sep 4 1986

At some point, possibly post code-complete, we should go back and normalize dates across the application.